docs_src/microservices/application/ApplicationServiceAPI.md

```diff
@@ -294,9 +293,9 @@ type FunctionPipeline struct {
 }
 ```
 
-### SetFunctionsPipeline
+### SetDefaultFunctionsPipeline
 
-`SetFunctionsPipeline(transforms ...AppFunction) error`
+`SetDefaultFunctionsPipeline(transforms ...AppFunction) error`
 
```

Comment:
Remove, not replace since SetDefaultFunctionsPipeline is already below.

Reply:
yep I have now fixed this